File: 485-MaxConsecutiveOnes/485-MaxConsecutiveOnes.java
// Last updated: 8/1/2025, 10:36:44 PM
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int count=0;
        int max=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]==1){
                count++;
            }else{
                max=Math.max(count,max);
                count=0;
            }
        }
        max=Math.max(count,max);
        return max;
    }
}

File: 485-MaxConsecutiveOnes/485-MaxConsecutiveOnes.py
# Last updated: 8/1/2025, 10:36:51 PM
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
       count=0
       max_count=0
       for i in nums:
         if i==1:
            count+=1
            if max_count<count:
              max_count=count
         else:
            count=0   
       return max_count